app/controllers/LdapController.java

```diff
@@ -101,7 +101,7 @@ public Result index() {
         final List<RoleResponse> roles = Lists.newArrayList(rolesService.loadAll());
         final Set<String> selectedAdditionalGroups = ldapSettings != null && ldapSettings.getAdditionalDefaultGroups() != null ? ldapSettings.getAdditionalDefaultGroups() : Collections.<String>emptySet();
         return ok(views.html.system.ldap.index.render(currentUser(), breadcrumbs(), ldapSettingsForm, roles,
-                                                      selectedAdditionalGroups));
+                                                      selectedAdditionalGroups, ldapSettings.getGroupMapping()));
```

Comment:
If no LDAP settings have been created, this leads to a NullPointerException in the web interface (`app/controllers/LdapController.java:104`).